porting/nimble/include/os/os_mbuf.h

```diff
@@ -413,6 +413,21 @@ struct os_mbuf *os_mbuf_off(const struct os_mbuf *om, int off,
  */
 int os_mbuf_copydata(const struct os_mbuf *m, int off, int len, void *dst);
 
+/**
+ * @brief Calculates the length of an mbuf chain.
+ *
+ * Calculates the length of an mbuf chain.  If the mbuf contains a packet
+ * header, you should use `OS_MBUF_PKTLEN()` as a more efficient alternative to
+ * this function.
+ *
+ * @param om                    The mbuf to measure.
+ *
+ * @return                      The length, in bytes, of the provided mbuf
+ *                                  chain.
+ */
+uint16_t os_mbuf_len(const struct os_mbuf *om);
+/**
```

Comment:
bogus comment start

This isn't in core... did you copy code fragments manually instead of simply copying whole file?

Reply:
I copied whole contents of corresponding files, except of includes and adjusted some defined values to match.

This method is here:
https://github.com/apache/mynewt-core/blob/c5d9068cd472f32f0afd89594038a188d9ddf50d/kernel/os/include/os/os_mbuf.h#L429

Reply:
so before copying I was checking diff, to see what is added and what might cause errors

Reply:
by this I mean extra /** added after function declaration, which is causing issues due to double comment start (aka comment in comment)

Reply:
Ok, I remember now: I was indeed copying fragments indicated by diff - to prevent conflict between types, for example `ble_npl_event` and `os_eventq`. So yes, this is caused by copying fragments.